docs/tutorials/step-2.md

```diff
@@ -24,25 +24,25 @@ in the previous paragraph is implemented as follows:
 open Test_lib
 open Report
 
-let exercise_1 = 
-  Section ([ Text "The identity of 0 is 0." ],
-           test_function_1_against_solution
-             [%ty: int -> int] "identity"
-             ~gen:0 [0]
-          )
+let exercise_1 =
+  grade_function_1_against_solution
+    [%ty: int -> int] (* Type of the tested function *)
+    "identity"        (* identifier of the tested function *)
```

Comment:
There is a heterogeneous usage of uppercase and lowercase styles in the comments.